lib/src/main/java/com/auth0/jwt/algorithms/Algorithm.java

```diff
@@ -365,20 +366,20 @@ public String toString() {
     /**
      * Sign the given content using this Algorithm instance.
      *
-     * @param headerBytes an array of bytes representing the base64 encoded header content to be verified against the signature.
+     * @param headerBytes  an array of bytes representing the base64 encoded header content to be verified against the signature.
```

Comment:
```suggestion
     * @param headerBytes an array of bytes representing the base64 encoded header content to be verified against the signature.
```

Reply:
white spaces are being added because the class wasn't properly formatted following the project defaults (IDE default settings).